lumigator/python/mzai/backend/backend/tests/api/routes/test_api_workflows.py

```diff
@@ -5,21 +5,16 @@
 from fastapi.testclient import TestClient
 from schemas.datasets import DatasetFormat, DatasetResponse
 
+from backend.main import app
+
 
 # First test waits and polls up to 10 times for the real backend to be up.
+@app.on_event("startup")
```

Comment:
Injecting a real instance of the app on startup, per discussion here https://github.com/mozilla-ai/lumigator/pull/290#discussion_r1808768658